Dapper.Tests/PerformanceTests.cs

```diff
@@ -129,49 +131,51 @@ public void Run(int iterations)
                 Try(() =>
                 {
                     var l2scontext1 = GetL2SContext(connection);
-                    tests.Add(id => l2scontext1.Posts.First(p => p.Id == id), "Linq 2 SQL");
+                    tests.Add(id => l2scontext1.Posts.First(p => p.Id > id), "Linq 2 SQL");

```

Comment:
Can you explain these changes? Not getting the item asked for doesn't make any sense...or I'm just crazy.